<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClientWidget</class>
 <widget class="QWidget" name="TcpClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpClientWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QListWidget" name="contentListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_setting" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>601</width>
        <height>141</height>
       </rect>
      </property>
      <widget class="QPushButton" name="enterBtn">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>进入聊天室</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userNameLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Q君s</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverIPLineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="portLineEdit">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>12345</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sendBtn">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="sendtextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>501</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
